resources/scripts.js

```diff
@@ -488,6 +477,11 @@ const PRIMARYLOSE = `7 was rolled, ${playerPoint} missed and you lose`;
 const INVALIDBET = 'Please make a valid place bet';
 const BANKRUPT = `You are bankrupt! Please choose 'New Game' from the menu to play again`;
 const PLACEONPOINT = 'You cannot place on the current point';
+const SECONDARYLOSE = 'The place bets have lost';
+const PLACEWIN = `Place bet on ${playerPoint}`;
+const COMEWIN = 'The last come bet has won!';
+const COMELOSE = 'The last come bet has lost';
+const COMESET = `The last come bet is placed on ${playerPoint}`
```

Comment:
Does this work? I have never really seen a variable used in a string constant before.

Reply:
Changed the string constants to not use variables. Instead, string constants are rewritten so if a variable is needed, it would appear after the string